documentation/api/query/v4/query.markdown

```diff
@@ -95,6 +95,19 @@ Most PuppetDB query endpoints support paged results via a set of shared URL para
 
 All queries return data with a content type of `application/json`. Each endpoint's page describes the format of its return data.
 
+### Rich data
+
+Puppet 6 supports
+[rich_data](https://github.com/puppetlabs/puppet-specifications/blob/master/language/types_values_variables.md#richdata)
+types like Timestamp and SemVer, and enables rich data by default.
+When rich data is enabled, readable string representations of rich
+data values may appear in the report resource event `old_value` and
```

Comment:
Maybe this should be "will appear"? "may appear" may give them the impression that they have some way of controlling it

Reply:
Hmm, well I went with "may" because even if rich data's enabled, most people won't see any values right now?  i.e. just because it's turned on doesn't mean it'll show up in any given place, and the other stuff will look the way it always has, or is that also not quite right?